File: Life/Domain/Cell.cs
namespace Domain {
    public class Cell {
        private readonly bool isAlive;

        public Cell(bool isAlive) {
            this.isAlive = isAlive;
        }


        public bool IsDead {
            get { return !isAlive; }
        }

        public bool IsAlive {
            get { return isAlive; }
        }
    }
}

File: life-java/src/main/java/ru/agiledays/life/server/domain/Cell.java
package ru.agiledays.life.server.domain;

public class Cell {
        private final boolean isAlive;
        
        public Cell(final boolean isAlive) {
            this.isAlive = isAlive;
        }
        
        public boolean isDead() {
            return !isAlive;
        }
        
        public boolean isAlive() {
            return isAlive;
        }
}
